infra/modules/service/main.tf

```diff
@@ -37,7 +37,8 @@ locals {
     { name : "DEPLOY_GITHUB_SHA", value : data.external.deploy_github_sha.result.value },
     # TODO: https://github.com/HHS/simpler-grants-gov/issues/3177
     # { name : "DEPLOY_GITHUB_REF", value : data.external.deploy_github_ref.result.value },
-    { name : "DEPLOY_WHOAMI", value : data.external.whoami.result.value }
+    { name : "DEPLOY_WHOAMI", value : data.external.whoami.result.value },
+    { name : "EXPORT_OPP_DATA_FILE_PATH", value : "${aws_s3_bucket.general_purpose.bucket_regional_domain_name}/public-extracts/" }
```

Comment:
@coilysiren - just confirming this approach works for you

Reply:
Hi @coilysiren, would you be able to leave a review please?

Reply:
Huh... why didn't I get notifications about this earlier. I'm usually on top of this sort of thing :/

Reply:
So this isn't quite what we want - we want to avoid putting API specific logic into `infra/modules/whatever` unless it goes into its own file